pkg/server/update_runtime_config_test.go

```diff
@@ -51,7 +51,7 @@ func TestUpdateRuntimeConfig(t *testing.T) {
 	},
 	]
 }`
-		testCIDR = "10.0.0.0/24, 2001:4860:4860::8888/32"
+		testCIDR = "10.0.0.0/24, 2001:4860:4860::/64"
```

Comment:
2001:4860:4860::8888/32 is not a valid subnet, the valid subnet is 2001:4860::/32 and the cni fails with the following error,
```
emote_runtime.go:130] RunPodSandbox from runtime service failed: rpc error: code = Unknown desc = failed to setup network for sandbox "d218bbc3da33c4dbf22d6098191831da5474ce4cfabeb9f00441b09853cd671a": invalid range set 1: Network has host bits set
```

It's also better to use common masks for the tests, in IPv6 it is /64, to avoid unexpected issues with overlapping IPs

Reply:
> invalid range set 1: Network has host bits set

That's what I was wondering about.. thx.  I wonder why it was originally pushed with google's DNS lookup IP.. that resolves to dns.google... A CIDR test app I tried.. seemed to be fine with with what we had it.

Reply:
example localhost setup: 
https://docs.projectcalico.org/reference/cni-plugin/configuration